NAMESPACE

```diff
@@ -3,14 +3,12 @@
 export(calcNetTrade)
 export(calcNetTradeValue)
 export(calcPrice)
-export(calcTimeSeriesStats)
 export(calc_CES_marginals)
 export(calc_regionSubset_sums)
 export(checkVsCalibData)
 export(colorScenConf)
 export(compareCalibrationTargets)
 export(compareScenConf)
-export(compareScenarios2)
```

Comment:
again, I think we should keep `remind2::compareScenarios2()` as long as remindmodel `master` calls it. But maybe we should decide more generally if we strive for compatibility of the latest remindmodel release with the most recent `remind2` version. Given my experience in ENGAGE, ELEVATE and NGFS, I would strongly opt for making sure that work, because most projects seems to start from the release and then add some changes to it that require adding stuff to remind2. The most convenient way to avoid divergence of project work and model development is to just push that to the `remind2` master and work with that. But if compareScenarios2 is broken then, this workflow impossible – which I think creates more problems and unnecessary double-work compared to making sure we allow for backwards compatibility. Maybe we should discuss that more generally within the remind team.

Reply:
Personally, I am not a fan of coding for hypothetical causes. I have a hard time wrapping my head around a specific use case where you cannot simply merge the latest REMIND and remind2 into your project  branches (unless you never want to merge your work back into the master branches). 

On the other hand, I am very familiar with the technological debt created by writing and maintaining code for all eventualities.  So I personally would recommend to merge this and offer help to any project that actually ends up struggling with this on a case by case basis. 

As I want this to be merged, I am bringing back `compareScenarios2` for now, but will raise this topic with RSE.

Reply:
It is not hypothetical. It is the usual workflow in every single project I'm involved in (NGFS, ELEVATE, COMMITTED) and I guess everywhere when you want to use a validated REMIND version. The current `develop` is neither validated nor stable and I would not recommend to use it for project work. But yes, please discuss that within RSE and I think we should discuss that in the REMIND meeting.